admin/database/postgres/migrations/0034.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE project_access_request (
+    id UUID NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    email TEXT NOT NULL,
```

Comment:
I know this might have UX implications, but it would be great if we could require a user to log in before requesting access, and then store the `user_id` here instead.

By requiring a user to log in, we:
- Avoid sending lots of spam access request emails
- Can better mitigate data harvesting attacks
- Safeguard against pages like "Request access to Customer X on Rill" showing up in search engine results.

Reply:
We already require login.